docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,213 @@
+  </video>
+</div>
+**Current Blueprints available**   
+On the monoDrive Editor look for the Content Browser, from the folders select Vehicles. You will find the current models available.
+
+When selecting a model programmatically (i.e. using any client) you can assign the model to use for the ego vehicle to be random from the vehicle blueprints, you can find it on your Editor folder under:  
+`Config/ConfigRandomVehicleSettings.ini`   
+
+```
+[VehicleBlueprints]
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_01.sedan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/sedan_monoDrive_02.sedan_monoDrive_02_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/SUV_monoDrive_01.SUV_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/truck_monoDrive_01.truck_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/subcompact_monoDrive_01.subcompact_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/minivan_monoDrive_01.minivan_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/coupe_monoDrive_01.coupe_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C'
+VehicleBlueprint=Blueprint'/Game/Vehicles/compact_monoDrive_01.compact_monoDrive_01_C'
+```
+**Note:** If you want to add your own vehicle model, add the path to the blueprint to this file.  
+
+## Change the vehicle's color
+<div class="img_container">
+    <video width=650px height=340px muted controls autoplay loop>
+        <source src="http://cdn.monodrive.io/readthedocs/vehicle_color.mp4" type="video/mp4">
+    </video>
+</div> 
+
+**Current colors available**   
+On the monoDrive Editor look for the Content Browser. From the folders select Vehicles and then select CarPaint, you will find the current colors availables for any car.
+
+```
+Carpaint_BabyBlue
+Carpaint_Beige
+Carpaint_Black
+Carpaint_Copper
+Carpaint_DarkGrey
+Carpaint_Gold
+Carpaint_LightBlue
+Carpaint_NavyBlue
+Carpaint_OliveGreen
+Carpaint_Red
+Carpaint_Silver
+Carpaint_White
+```
+When selecting a color programmatically (i.e. using any client) you can assign the color to use for the ego vehicle using the `vehicle.json` with the `body` tag. 
+```
+"body": {
+    "color": "Carpaint_White.Carpaint_White",
+    "type": "/Game/Vehicles/crossover_monoDrive_01.crossover_monoDrive_01_C"
+  }
+```
+## PID Speed Controllers
+### Follower PID
+Each vehicle has a pre-defined PID parameters to control how closely and accurate follow others vehicles. You can tune this parameters.
+
+### Speed Maintain PID
+Each vehicle has a pre-defined PID parameters to control how closely and accurate maintains the desired speed.
+
+## Vehicle Dynamics
+Using the monoDrive vehicles you have access to the Mechanical Setup to control the drivetrain type (4WD,FDW,RDW,etc), the maximum RPM and damping, the transmission's type, as well of the gear ratio if not automatic transmission, you can also set suspension settings and more.
```

Comment:
replace "you" with "the user has"
Using the monoDrive vehicles you have access to the Mechanical Setup to control the drivetrain type (4WD,FDW,RDW,etc), the maximum RPM and damping, and the transmission's type.  The gear ratio may be changed to automatic transmission, and the user can set suspension settings and more.

Reply:
Here's what Stephen suggested in slack : monoDrive Scenario Vehicles are built on PhysX vehicles which are customizable out of the box. All of the PhysX mechanical settings can be cusomized in the editor to match your vehicle such as the suspension, drivetrain type, transmission gearing, suspension parameters, and much more.